pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+Huge thanks to [Krishna Kumar](https://github.com/Krishks369) and [Shreyansh Jain](https://github.com/sudoshreyansh) that helped in research related to the implementation :heart:
+
+If you want to learn more about becoming a host of the meeting, or other things related to meetings, get familiar with our [dedicated meetings FAQ](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).
+
+Oh, and I almost forgot, look at this :point_down: :star: 
+All [info about AsyncAPI-related meetings](https://www.asyncapi.com/community/meetings) in one beatifull place :heart:
+
+<Figure
+  src="/img/posts/2022Q1-summary/meetings.webp"
+  className="text-center"
+  caption="View of dedicated community/meetings page on the AsyncAPI website."
+/>
+
+## Upcoming 2.4 and 3.0 releases of the spec
+
+We released a new [2.4.0 version of AsyncAPI Specification](https://github.com/asyncapi/spec/releases/tag/v2.4.0). There are many good features, like the possibility to provide the `id` of the Message unique for the entire AsyncAPI file or the ability to specify security valid only for a given operation. You definitely need to read [release notes](https://www.asyncapi.com/blog/release-notes-2.4.0).
+
+Huge shout out to [Sergio Moya](https://twitter.com/smoyac), who was a release coordinator for 2.4 release and pushed all the doors to make sure the release could go out.
+
+2.4.0 was released, but this doesn't block us from working on something much bigger, 3.0.0 version of the specification :rocket:
+
+If you do not know yet about it, I highly recommend you to have a look at [this article](https://www.asyncapi.com/blog/async-api-spec-3.0-release) from [Harsh Mishra](https://www.linkedin.com/in/harshcasper/).
+
+Please join these efforts, as there is much work ahead of us, but not many hands doing actual work. We meet every two weeks to sync the work efforts and discuss important topics. Feel free to join :pray:
+
+I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for [his regular updates on specification-related efforts](https://gist.github.com/smoya)
+
+## Contributors onboarding initiatives
+
+One of [goals for community-building efforts for 2022](https://github.com/asyncapi/community/discussions/193) was to involve AsyncAPI Initiative as a mentoring organization in as many coding (but not only) events. We have many maintainers here who like to help first-time contributors land their first pull requests in an open and relaxed environment.
+
+### Google Summer of Code
+
+We did not start well. We were not accepted for [Google Summer of Code](https://summerofcode.withgoogle.com). Don't ask me why. I really do not know. I wish I could have a clear answer from them about what failed on our end :sad:
```

Comment:
```suggestion
We did not fare well; we were not accepted for [Google Summer of Code 2022](https://summerofcode.withgoogle.com). Don't ask me why. I really do not know. I wish I could have a clear answer from them about what failed on our end. :sad:
```
I just have 1 thought.. the word **fail** is very strong and we may want to think about a more accurate way to describe what (probably?) happened.  What I mean is that the AsyncAPI GSoC application didn't necessarily fail, it could simply have been in tough competition due to the **amount** of OSS orgs that applied. 

We already know that a humongous company like Google easily gets BASICALLY THE WHOLE WORLD to apply to their programs, so if they got 2000+ applications and decided to only accept whatever number of applicants in 2022... odds are we were only one of a gazillion applications.

I would consider re-phrasing this the first part of this section so that it doesn't influence the community to feel too negative about getting a no this year.

Reply:
- `fare well` - doesn't it mean something like good bye? what I meant by `we did not start well` was that we did not start well, like a false start in a race :) 
- I modified a bit the `fail` part but left the word. I just made it more personal, that I think it is my failure. If in one place we say that someone succeeded with something, it is ok to also personally admit that I think I failed with GSoC application. If I blame others, I will not improve. Now I know that next year we need to work on official application in a group, to make sure we write it better. Hope you know what I mean